desktop-app/app/menu.js

```diff
@@ -340,6 +342,19 @@ export default class MenuBuilder {
             this.mainWindow.toggleDevTools();
           },
         },
+        {
+          label: 'Show Status Bar',
+          type: 'checkbox',
+          checked: statusBarSettings.getVisibility(),
+          click: () => {
+            const newVisibility = !statusBarSettings.getVisibility();
+            statusBarSettings.setVisibility(newVisibility);
+            this.mainWindow.webContents.send(
+              STATUS_BAR_VISIBILITY_CHANGE,
+              newVisibility
+            );
+          },
+        },
```

Comment:
@manojVivek should we consider starting moving these actions to constants so we don't repeat them?

Reply:
@falecci Yeah agreed, it is high time we should do it. And maybe simply it by getting rid of having two different menu setup for Mac and Windows.